tensorboard/webapp/widgets/data_table/data_table_component.ts

```diff
@@ -256,11 +256,13 @@ export class DataTableComponent implements OnDestroy, AfterContentInit {
       x: rect.x + rect.width,
       y: rect.y + rect.height,
     });
+    this.columnSelector.activate();
```

Comment:
I don't love this being here but I can't think of a better solution that isn't janky.

Reply:
We are on the same page here.